addons/website_sale/models/sale_order_line.py

```diff
@@ -46,6 +46,8 @@ def _get_sale_order_line_multiline_description_sale(self):
         return description
 
     def get_description_following_lines(self):
+        if self.env.context.get('lang') != self.env['ir.http'].session_info().get('bundle_params').get('lang'):
+            self._compute_name()
```

Comment:
`get_description_following_lines` is called from multiples places in Odoo and I'd like to avoid checking this condition each time. Could you move your diff to the controller of `/shop/cart`?